components/Mappage/SideBar.tsx

```diff
@@ -1,44 +1,39 @@
 import { useState } from 'react'
 import Card from './Card'
 import styles from '@/styles/componentStyles/SideBar.module.css'
+import Image from 'next/image';
+
 
 function SideBar(props: any) {
 
+  console.log(props.placesInfo)
+
   return (
 
     <div className={styles.infoSidebar}>
-      {/* <Tabs.Root defaultValue="tab1" orientation="vertical">
         <div className={styles.btnContainer}>
-          <Tabs.List className={styles.tabLists} aria-label="All trips">
-            <div className={styles.triggerContainer}>
-              <Tabs.Trigger className={`${styles.tabLabel} ${styles.leftTab}`} value="newLoctaionsTab">New locations</Tabs.Trigger>
-              <Tabs.Trigger className={styles.tabLabel} value="savedLocationsTab">Saved Locations</Tabs.Trigger>
-              <div className={styles.greyBG}></div>
-            </div>
-          </Tabs.List>
           <button className={styles.closeBtn} onClick={() => props.setShowInfo(false)}>
             Close
           </button>
         </div>
-        <Tabs.Content value="newLoctaionsTab">
           {props.placesInfo ? props.placesInfo.map((place: any, index: number) => (
             <Card
               key={index}
+              imageUrl={place.imageUrl}
```

Comment:
I see the rest of these are optionally chained. The only reason you would need to optionally chain here is if `place` could ever be undefined. Is this a possibility? 
If not, we should remove all the optional chaining

Reply:
I can kill this if needed, however, my thought behind this is if it is missing that property, ie: shortDescription or imageUrl or rating it wouldn't just crash. Is my understanding incorrect?

Reply:
No, optionally chaining is for when you are trying to access a property on a value that is literally `undefined`. 
If `place` is just an object that just so happens to not have the property of `imageUrl`, then `place.imageUrl` is just going to be `undefined`. 
BUT, if `place` is `undefined`, and you try to access a property `imageUrl` then you are literally trying to do `undefined.imageUrl` and `undefined` has NO properties to access in this way and therefore would throw the error "trying to access property 'imageUrl' of undefined"

Optionally chaining is only used when the object you are accessing the property of COULD be undefined

Reply:
Ahhhhh okay, got it going to remove the optional chaining